packages/local_auth/lib/local_auth.dart

```diff
@@ -72,4 +74,40 @@ class LocalAuthentication {
     }
     return await _channel.invokeMethod('authenticateWithBiometrics', args);
   }
+
+  /// Returns true or false if biometrics are available
```

Comment:
Fix comment. This checks if device is capable of checking biometrics not whether biometrics is available. You can have a device with biometric capability but not enrolled in any. In the latter case, this returns true (but no biometrics is available for the app to check).